backend/app/db/models/case.py

```diff
@@ -0,0 +1,100 @@
+import enum
+from sqlalchemy import (
+    Column,
+    Enum,
+    Integer,
+    Text,
+    ForeignKey,
+    Boolean,
+)
+
+from app.db.session import Base
+from . import Sector, Geography
+
+
+class LitBody(Base):  # noqa: D101
+    """Database model for a Litigation body /  legal entity."""
+
+    __tablename__ = "lit_body"
+
+    id = Column(Integer, primary_key=True)
+    name = Column(Text, unique=True, nullable=False)
+    description = Column(Text)
+
+
+class CaseStatus(enum.Enum):
```

Comment:
Subclassing `str` can make these enums more convenient to deal with:

```suggestion
class CaseStatus(str, enum.Enum):
```
e.g. after subclassing `str` `f"{CaseStatus.OPEN}"` would produce `"Open"` rather than `"CaseStatus.OPEN"`, and direct comparison to the values coming out of the CSV files would be possible (e.g. `CaseStatus.OPEN == "Open"` would return `True`)

Reply:
Ooo - going to take a look at this now